CruCentralCoast/Resources/ResourcesTableViewCollectionViewCell.swift

```diff
@@ -30,56 +34,64 @@ class ResourcesTableViewCollectionViewCell: UICollectionViewCell {
         self.tableView.registerCell(AudioResourcesCell.self)
         self.tableView.registerCell(VideosResourcesCell.self)
         self.tableView.registerCell(ArticlesResourcesCell.self)
+        
+        self.tableView.refreshControl = UIRefreshControl()
+        self.tableView.refreshControl?.addTarget(self, action: #selector(self.refreshResources(_:)), for: .valueChanged)
+    }
+    
+    @objc private func refreshResources(_ refreshControl: UIRefreshControl) {
+        ResourceManager.instance.refreshResources(ofType: self.type) { resources in
+            UIView.animate(withDuration: 0.3) {
```

Comment:
Why is this animated? What gets animated?

Reply:
!!! you just fixed this stupid UI bug i was struggling with for way too long. I have no clue why i put this inside a 
```swift
UIView.animate(withDuration:)
```
block. But because i did, i was getting this weird effect where the activity indicator was overlapping the tableview content. But getting rid of the call to ```UIView.animate()``` just fixed it! thank you!

Reply:
🙌